wikichange/.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
what are these rules used for? formatting?

Reply:
I think they are defaults created by the cli. They should only effect linting which is just how IDEs highlight code.